docs/source/data_ingestion_and_preparation/connector_support.md

```diff
@@ -0,0 +1,7 @@
+# Writing Docsets to External Data Stores
+
+Sycamore allows the ability to write Docsets out to external databases. This is needed to ensure the value gained from the ETL transfroms run in Sycamore can be in a usable format. Currently, Pinecone, Weaviate, Elasticsearch, DuckDB, and Opensearch are supported as external databases. Users can access this from a unified write interface, from where they can specify their databases and the respective connection and write arguments.
```

Comment:
DocSet

Reply:
"Sycamore allows the ability to write Docsets out to external databases. This is needed to ensure the value gained from the ETL transfroms run in Sycamore can be in a usable format." Can we change this to "A final step in a Sycamore processing job is to load the data into a target database for use in your application. This could be a combination of a vector index and term-based index, and includes the enriched metadata from the job..."

Reply:
"external" -> "target"